src/main/java/ch/saunah/saunahbackend/service/MailService.java

```diff
@@ -45,36 +50,141 @@ public void sendUserActivationMail(String email, String verificationId) {
+    public void sendUserApprovedBookingMail(String email, Booking booking, BookingPrice bookingPrice, BookingSauna bookingSauna) {
+        String mailText = "<p>Ihre Buchung wurde genehmigt. Hier sehen sie die Buchung:</p>";
+        messageUserSetup(email, mailText, booking, bookingPrice, bookingSauna);
+    }
+
+    /**
+     * This method sends a mail which informs the user that his booking has been canceled.
+     *
+     * @param email        The email of the user
+     * @param booking      The Booking info
+     * @param bookingPrice The bookingPrice info
+     * @param bookingSauna The bookingSauna info
+     */
+    public void sendUserCanceledBookingMail(String email, Booking booking, BookingPrice bookingPrice, BookingSauna bookingSauna) {
+        String mailText = "<p>Ihr Buchung wurde storniert. Hier sehen sie die Buchung:</p>";
+        messageUserSetup(email, mailText, booking, bookingPrice, bookingSauna);
+    }
+
+    private void messageUserSetup(String email, String mailText, Booking booking, BookingPrice bookingPrice, BookingSauna bookingSauna) {
+        try {
+            MimeMessage mimeMessage = javaMailSender.createMimeMessage();
+            MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, "utf-8");
+            helper.setFrom(new InternetAddress(senderEmail, senderName));
+            helper.setTo(email);
+            helper.setSubject("Booking Info SauNah");
+            helper.setText(mailText + bookingInfo(booking, bookingPrice, bookingSauna), true);
+            javaMailSender.send(mimeMessage);
+        } catch (MessagingException | MailException | UnsupportedEncodingException exception) {
             System.err.printf(DEFAULT_MAIL_ERROR_MESSAGE, exception.getMessage());
         }
     }
+
+    /**
+     * This method sends a mail to all admins that a new booking has been created.
+     *
+     * @param adminEmailList The email list of the all admins
+     * @param booking        The Booking info
+     */
+    public void sendAdminOpenedBookingMail(List<User> adminEmailList, Booking booking) {
+        for (User admin : adminEmailList) {
+            try {
+                MimeMessage mimeMessage = javaMailSender.createMimeMessage();
+                MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, "utf-8");
+                helper.setFrom(new InternetAddress(senderEmail, senderName));
+                helper.setTo(admin.getEmail());
+                helper.setSubject("Booking Info SauNah");
+                helper.setText("<p>Eine neue Buchung wurde eröffnet. Hier sehen sie die Buchung:<h1>" +
+                    "<br><a href=\"" + frontendBaseUrl + "/bookings/" + booking.getId() + "/" + "\">zur Buchung</a></p>", true);
+                javaMailSender.send(mimeMessage);
+            } catch (MessagingException | MailException | UnsupportedEncodingException exception) {
+                System.err.printf(DEFAULT_MAIL_ERROR_MESSAGE, exception.getMessage());
+            }
+        }
```

Comment:
i would change, that this method only sends to the user which was passed in the method argument. I would use the for loop in the other class where this method is called.